examples/with-scss/razzle.config.js

```diff
@@ -0,0 +1,77 @@
+// Based on: https://gist.github.com/jaredpalmer/0a91a7bd354b875b913c74f4b16125f7
+
+const autoprefixer = require('autoprefixer')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
```

Comment:
Shouldn't we use [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) ?